tex/rep-theory/rep-alg.tex

```diff
@@ -330,12 +330,12 @@ \section{Irreducible and indecomposable representations}
 	$a \cdot w \in W$.
 	In other words, this subspace is invariant under actions by $A$.
 \end{definition}
-Thus if $V \neq W_1 \oplus W_2$ for representations $W_1$, $W_2$
-then $W_1$ and $W_2$ are subrepresentations.
+Thus if $V = W_1 \oplus W_2$ for representations $W_1$, $W_2$
+then $W_1$ and $W_2$ are subrepresentations of $V$.
```

Comment:
How about adding here a sentence like: "However the converse is not true in general."

Reply:
What would "converse" mean here?

Reply:
I mean, `W_1` being a subrepresentation of `V` doesn't imply that there's another subrepresentation `W_2` such that `V = W_1 ⊕ W_2`. (Not the exact converse though.) This is the difference between ireducability and indecomposability, one of the main points of the whole chapter, so I thought it should be emphasized more a bit.

Reply:
Ah I see. I think this is mentioned explicitly about half a page later, between the first example and the exercise after that. This timing is on purpose since I think it's better to have the notion of irreducible/indecomposable plus an example first before mentioning this.